cfpb/cfgov-refresh
# Use ShareableRoutablePageMixin for regulation pages

This change uses `ShareableRoutablePageMixin` added to wagtail-sharing in https://github.com/cfpb/wagtail-sharing/pull/27 to get around the issue described in https://github.com/cfpb/wagtail-sharing/issues/26 as it effects new regulation pages that do not have approved content and are not ready to publish.

## Testing

1. Unpublish the regulation page for Regulation B: http://localhost:8000/admin/pages/11766/edit
2. Visit the regulation page for Regulation B at your local sharing URL in Chrome and attempt to preview, observe a 404: http://content.localhost:8000/policy-compliance/rulemaking/regulations/1003/
3. Install wagtail-sharing from https://github.com/cfpb/wagtail-sharing/pull/27
4. Visit the regulation page for Regulation B at your local sharing URL in Chrome and attempt to preview, observe the sharing banner and that you can now preview the regulation: http://content.localhost:8000/policy-compliance/rulemaking/regulations/1003/
5. Visit the regular URL for Regulation B and observe a 404: http://localhost:8000/policy-compliance/rulemaking/regulations/1003/

## Checklist
